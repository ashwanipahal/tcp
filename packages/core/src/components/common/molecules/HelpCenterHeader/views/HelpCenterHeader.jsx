import React from 'react';
import PropTypes from 'prop-types';
import withStyles from '@tcp/core/src/components/common/hoc/withStyles';
import { Image } from '@tcp/core/src/components/common/atoms';
import { getIconPath, getLocator } from '@tcp/core/src/utils';
import style from '../styles/HelpCenterHeader.style';

const HelpCenterHeader = ({ className, labels }) => (
  <div className={className}>
    <div className="help-center-icon">
      <Image src={getIconPath('circle-info-blue')} alt="Store Icon" />
    </div>
    <div className="help-center">
      <div className="help-center-title">
        <h4 className="help-center-title__text" data-locator={getLocator('')}>
          {labels.helpcenter_pageHeader || 'HELP CENTER'}
        </h4>
      </div>
    </div>
  </div>
);

HelpCenterHeader.propTypes = {
  className: PropTypes.string.isRequired,
  labels: PropTypes.shape({
    helpcenter_pageHeader: PropTypes.string,
  }),
};

HelpCenterHeader.defaultProps = {
  labels: {
    helpcenter_pageHeader: 'HELP CENTER',
  },
};

export default withStyles(HelpCenterHeader, style);
