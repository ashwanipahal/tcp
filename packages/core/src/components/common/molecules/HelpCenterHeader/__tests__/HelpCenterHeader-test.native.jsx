import React from 'react';
import { shallow } from 'enzyme';
import HelpCenterHeader from '../views/HelpCenterHeader.native';

describe('HelpCenterHeader component', () => {
  it('HelpCenterHeader component renders correctly without props', () => {
    const component = shallow(<HelpCenterHeader />);
    expect(component).toMatchSnapshot();
  });
});
