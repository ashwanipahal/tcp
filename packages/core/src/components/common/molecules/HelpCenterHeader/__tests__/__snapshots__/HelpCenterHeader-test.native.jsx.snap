// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelpCenterHeader component HelpCenterHeader component renders correctly without props 1`] = `<Styled(View) />`;
